Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -57,7 +57,7 @@ public class HttpListenerHost : IHttpServer, IDisposable
             _appHost = applicationHost;
             _logger = logger;
             _config = config;
-            _defaultRedirectPath = configuration["HttpListenerHost:DefaultRedirectPath"];
+            _defaultRedirectPath = configuration["HttpListenerHost_DefaultRedirectPath"];
```

Comment:
Don't they use `:` as the separator between levels for `Configuration`? Do we want a flat list instead?

Reply:
How would the env var look like with multiple levels?

Reply:
It replaces `:` with "__" (two underscores).

So `ConnectionStrings__Main` for 
```json
{
  "ConnectionStrings": {
    "Main": "value here"
  }
}
```